src/storage/src/metrics/source/mysql.rs

```diff
@@ -7,21 +7,133 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-//! Metrics for Postgres.
+//! Metrics for MySQL.
 
-use mz_ore::metric;
-use mz_ore::metrics::{DeleteOnDropGauge, GaugeVec, GaugeVecExt, MetricsRegistry};
-use mz_repr::GlobalId;
-use prometheus::core::AtomicF64;
 use std::sync::Arc;
 use std::sync::Mutex;
 
+use prometheus::core::{AtomicF64, AtomicU64};
+
```

Comment:
nit: We group mz and external crates in a single section https://github.com/MaterializeInc/materialize/blob/main/doc/developer/style.md#imports

```suggestion
```